circle.yml

```diff
@@ -32,6 +32,7 @@ test:
   override:
     - ./scripts/circle-ci/run-circle-tests.sh:
         parallel: true
+        timeout: 900
```

Comment:
This was added to help debug Circle CI builds timing out with the default timeout of 600 seconds. There is definitely something going on here since our tests are running much slower (constant overhead of about 7-8 seconds for all Cassandra integration tests). I am still investigating the cause.